scrapinghub/client/collections.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
-import collections
 
 from six import string_types
+from six.moves import collections_abc
```

Comment:
It does not need to happen in this pull request, but it may be time to remove dependency on six altogether, since we are stopping Python 2 support.